rpc/README.md

```diff
@@ -4422,6 +4423,53 @@ Response
 }
 ```
 
+<a id="pool-estimate_fee_rate"></a>
+#### Method `estimate_fee_rate`
+* `estimate_fee_rate(target_to_be_committed)`
+    * `target_to_be_committed`: [`Uint64`](#type-uint64)
+* result: [`Uint64`](#type-uint64)
+
+Estimate fee rate for a transaction to be committed within target block number by using a simple strategy.
+
+Since CKB transaction confirmation involves a two-step process—1) propose and 2) commit, it is complex to
+predict the transaction fee accurately with the expectation that it will be included within a certain block height.
+
+This method relies on two assumptions and uses a simple strategy to estimate the transaction fee: 1) all transactions
+in the pool are waiting to be proposed, and 2) no new transactions will be added to the pool.
+
+In practice, this simple method should achieve good accuracy fee rate and running performance.
+
+###### Params
+
+* `target_to_be_committed` - The target block number to be committed, minimum value is 3 and maximum value is 131.
```

Comment:
> The rpc parameter target_to_be_committed is limited to 3~131, because the minimum value of the proposal window is 2, which is equivalent to predicting the height of current_tip + 1 ~ 128.

128+2=130